dapp/src/components/FuseDashboard/components/RewardUserForm/index.jsx

```diff
@@ -18,6 +23,12 @@ const RewardUserForm = ({ initialValues, setJoinBonus, setBackupBonus, setInvite
     }
   }
 
+  const initialValues = React.useMemo(() => ({
+    joinBonus: { ...get(plugins, 'joinBonus.joinInfo'), isActive: get(plugins, 'joinBonus.isActive') },
```

Comment:
you don't have mobx warnings here?
I learned it's problematic to use  observables `React.useMemo`
IMO we can just skip the useMemo here

Reply:
Yes, you're right.
good catch, thanks.